Java/BucketSort.java

```diff
@@ -0,0 +1,44 @@
+//Bucket Sort is an sorting algoritm in which we sort numbers by distributing among buckets.

```

Comment:
How is this logic of bucket sort? You are not maintaining buckets or hashing numbers according to you bucket size.

Seems like you implemented counting sort instead

Reply:
@YashSaxena9 I got bit confused with the concept. I have updated the the code. Please review.
Thanks